libs/core/tests/unit_tests/runnables/test_runnable.py

```diff
@@ -1020,6 +1020,115 @@ def test_configurable_alts_factory() -> None:
     assert fake_llm.with_config(configurable={"llm": "chat"}).invoke("...") == "b"
 
 
+def test_configurable_fields_prefix_keys() -> None:
+    fake_chat = FakeListChatModel(responses=["b"]).configurable_fields(
+        responses=ConfigurableFieldMultiOption(
+            id="responses",
+            name="Chat Responses",
+            options={
+                "hello": "A good morning to you!",
+                "bye": "See you later!",
+                "helpful": "How can I help you?",
+            },
+            default=["hello", "bye"],
+        ),
+        sleep=ConfigurableField(
```

Comment:
What happens if there's another configurable alternative that does not support `sleep`, but it's still makred as `is_shared`?

Reply:
In that case the other one doesnt use it (ie what happens already before this PR)